src/edu/washington/escience/myria/parallel/Server.java

```diff
@@ -1099,52 +1101,63 @@ public long createView(
   /**
    * Create a function and register it in the catalog
    */
-  public long createFunction(
-      final String functionName, final String functionDefinition, final Set<Integer> workers)
+  public String createFunction(
+      final String functionName,
```

Comment:
These parameter names should change, of course, if the encoding field names change.